beacon-chain/powchain/log_processing.go

```diff
@@ -455,17 +462,26 @@ func (s *Service) checkHeaderRange(start uint64, end uint64,
 	return nil
 }
 
-func (s *Service) checkForChainstart(blockHash [32]byte, blockNumber *big.Int, blockTime uint64) {
+func (s *Service) currentCountAndGenesisTime(blockTime uint64) (uint64, uint64) {
```

Comment:
```suggestion
func (s *Service) currentValidatorCountAndGenesisTime(blockTime uint64) (uint64, uint64) {
```

Reply:
this seems excessively verbose, especially given that this method is only used in this package.

Reply:
No problem. It's just that I saw the method name and needed to read the body to understand what does `Count` refer to.

Reply:
added in a comment